keyboards/tada68/keymaps/abishalom/keymap.c

```diff
@@ -0,0 +1,98 @@
+#include "tada68.h"
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BL 0
+#define _FL 1
+#define _NL 2
+
+#define _______ KC_TRNS
+#define SEMI_FN LT(_FL, KC_SCLN)
+
+
+//Tap Dance Declarations
+enum {
+  TD_WIN_LOCK = 0
+};
+
+//BL_BRTG - breathing mode. Would like to put in.
+
+//Tap Dance Definitions
+qk_tap_dance_action_t tap_dance_actions[] = {
+  [TD_WIN_LOCK]  = ACTION_TAP_DANCE_DOUBLE(MAGIC_NO_GUI, MAGIC_UNNO_GUI)
+};
+
+//
+////In Layer declaration, add tap dance item in place of a key code
+//TD(TD_WIN_LOCK)
+
+// Double Tap Fn button to get caps. FN + Tab to get to number layer -> Maybe switch to FN + space?. 
+// Press fn+GUI once to lock, double tap to unlock.
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /* Keymap _BL: (Base Layer) Default Layer
+   * ,----------------------------------------------------------------.
+   * |Esc | 1|  2|  3|  4|  5|  6|  7|  8|  9|  0|  -|  =|Backsp |~ ` |
+   * |----------------------------------------------------------------|
+   * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|  \  |Del |
+   * |----------------------------------------------------------------|
+   * |FNCAP|  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Enter  |PgUp  |
+   * |----------------------------------------------------------------|
+   * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift | Up|PgDn|
+   * |----------------------------------------------------------------|
+   * |Ctrl|Win |Alt |        Space          |Alt| FN|Ctrl|Lef|Dow|Rig |
+   * `----------------------------------------------------------------'
+   */
+[_BL] = KEYMAP_ANSI(
```

Comment:
Instances of `KEYMAP_ANSI` should be replaced with `LAYOUT_ansi`.

```suggestion
[_BL] = LAYOUT_ansi(
```